src/ign.cc

```diff
@@ -192,7 +192,6 @@ extern "C" IGNITION_GAZEBO_VISIBLE int runServer(const char *_sdfString,
       //   In Blueprint and Citadel, LogRecord will record to <path> if no
       //   --record-path is specified on command line.
       serverConfig.SetLogRecordPath(recordPathMod);
-      serverConfig.SetLogIgnoreSdfPath(true);
```

Comment:
The `logIgnoreSdfPath` in `ServerConfig.cc` has a comment that it should be deprecated in public APIs in Ignition-D (and removed in Ignition-E). Probably its corresponding functions `LogIgnoreSdfPath()` and `SetLogIgnoreSdfPath()` too. I'm not sure how to do that.

Reply:
The deprecation warnings are already there:

https://github.com/ignitionrobotics/ign-gazebo/blob/6332a61de6728b3aeb080fd587a04b569788f2a5/include/ignition/gazebo/ServerConfig.hh#L246-L254